app/src/main/java/com/example/sabonit/ProductActivity.java

```diff
@@ -70,20 +65,20 @@ protected void onCreate(Bundle savedInstanceState) {
         radioScentGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener()
         {
             public void onCheckedChanged(RadioGroup group, int checkedId) {
-                changeScent();
+                changeSmell();
```

Comment:
you have the permission to decide the appropriate name, although Im not sure it's the best decision.
just verify that we all use the name "smell" instead of "scent"

Reply:
i'll check it, the decision is to be smell everywhere in the code